src/port1.0/portpatch.tcl

```diff
@@ -50,7 +50,7 @@ options patch.asroot
 default patch.asroot no
 default patch.dir {${worksrcpath}}
 default patch.cmd {[portpatch::build_getpatchtype]}
-default patch.pre_args -p0
+default patch.pre_args "--forward -p0"
```

Comment:
Since `patch.pre_args` has always been used only for the `-p0` argument, it has not been uncommon for ports to override `patch.pre_args` when they need a different patch level such as `-p1`; there are currently 233 portfiles that do this, and three portfiles that erroneously use `patch.post_args` for this. So we either have to fix those 236 portfiles to use `patch.pre_args-replace -p0 -p1` (which perhaps arguably they should have been using all along) or we need to stash this flag somewhere else.

Reply:
Maybe we could use `patch.args` instead. Very few ports use it, and almost all of those that do are setting it to `-p1` and so should really be modifying `patch.pre_args` instead.